### ISSUE TITLE
[Feature Request] Only show released shows and movies in watchlist.

### ISSUE THREAD

**Issue by Spatula280** [Tikipeter/tikipeter.github.io#131]
Maybe I'm missing a setting but I looked everywhere. With FEN it used to only show movies that had already been released in the watchlist, even though I had movies like "Ready Player Two” in my Trakt watchlist it wouldn't show it. (Which I preferred)

When I switched to FEN lite, it showed everything. And honestly, it's it's a bit annoying sifting though to figure out what is available and what isn't.

I hope this can be achieved with ease.  Maybe a toggle switch in settings to show or not show. 
If not, I'll live with it.

**Comment by Tikipeter** [comment]
I should close this Issue, as using manners is one of the 3 rules for posting an Issue Request, but I want to add this back anyway.

Maybe next time use a "Thank you" at the end instead of "If not, I'll live with it".

**Comment by Spatula280** [comment]
I apologize, that wasn’t the intention of that statement.

It was more of “It’s not something that is mission critical. If it was a
quick and easy fix I would greatly appreciate it. If it’s not a quick easy
fix, then it’s something I can work around it.”

But yes, thank you for all you do.

**Comment by darbomusic** [comment]
This feature is already available. You sure you hide watched items from the widgets.

**Comment by Spatula280** [comment]
Darbomusic, 
Thank you for the comment,  I tried your suggestion and it doesn't solve the issue.  

The Trakt watchlist contains items you add to it, whether it has been released, in post-production, not even started filming, or some studios' long range plan (like avengers, avatar, fantastic beasts, etc).  

It's the shows/movies in the unreleased stage, that would be nice to be removed.  Currently when I go to my movies watchlist 1/2 of the list (almost 100 movies) haven't been released yet. Thus haven't been watched. So hiding ”watched item” wouldn't hide those.

Thanks for helping.
